React/Base/RCTModuleMethod.m

```diff
@@ -480,10 +480,10 @@ - (id)invokeWithBridge:(RCTBridge *)bridge
         expectedCount -= 2;
       }
 
-      RCTLogError(@"%@.%@ was called with %zd arguments, but expects %zd. \
-                  If you haven\'t changed this method yourself, this usually means that \
-                  your versions of the native code and JavaScript code are out of sync. \
-                  Updating both should make this error go away.",
+      RCTLogError(@"%@.%@ was called with %zd arguments but expects %zd arguments. "
+                  @"If you haven\'t changed this method yourself, this usually means that "
+                  @"your versions of the native code and JavaScript code are out of sync. "
+                  @"Updating both should make this error go away.",
```

Comment:
This is just to fix formatting. Before:

<img width="374" alt="screenshot 2017-01-27 11 25 28" src="https://cloud.githubusercontent.com/assets/346214/22361508/085e1ec6-e48d-11e6-8e08-afcac385a523.png">

After:

<img width="373" alt="screenshot 2017-01-27 11 28 02" src="https://cloud.githubusercontent.com/assets/346214/22361512/0d89efa6-e48d-11e6-875a-df4c777008e7.png">